### ISSUE TITLE
What would be the reason pool profit is blank in circle

### ISSUE THREAD

**Issue by IwasthefirstNerrdAlerrt** [RainbowMiner/RainbowMiner#2431]
Coin and pool are set up with wallet in both .config files, but totals are not showing up.  I'm wondering if because it is blank here, it is not showing up in my 1 hour, 24 hour, and 7 day profit.  Any suggestions on what to change to make this visible?
![image](https://github.com/RainbowMiner/RainbowMiner/assets/139030877/15d76490-4c63-461d-80bb-100e67eabf3c)

**Comment by RainbowMiner** [comment]
That would normally indicate (near) zero profitability. But I'll check with the Zpool API.

**Comment by RainbowMiner** [comment]
Ok, I have added the CRNC quote. It should arrive in your RainbowMiner instance in no time.

**Comment by IwasthefirstNerrdAlerrt** [comment]
I just noticed too, in accessing localhost,  zpool has been mining for the last 6 hours and the current hour is not showing any earnings. Could this be because the quote too?

**Comment by RainbowMiner** [comment]
Yes, probably. Also, it may be possible, that Zpool hasn't found a block (I doubt it though).

**Comment by whitindllc** [comment]
it might also be the payment.... how many crnc is equal to 0.00150000 BTC
